examples/ios/demo/RecaptchaDemo/RecaptchaDemoUITests/RecaptchaDemoUITests.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  RecaptchaDemoUITests.swift
+//  RecaptchaDemoUITests
+//
+//  Created by Walter Pinto on 2024-03-27.
```

Comment:
Find all of these and replace with a Google copyright/license.

Reply:
Done